cdap-docs/reference-manual/source/http-restful-api/metrics.rst

```diff
@@ -648,43 +731,139 @@ For aggregates, you cannot specify a time range. As an example, to return the to
 of input objects processed since the Application *CountRandom* was deployed, assuming that
 CDAP has not been stopped or restarted::
 
-  POST '<base-url>/metrics/query?context=namespace.default.app.CountRandom
+  POST '<base-url>/metrics/query?tag=namespace:default&tag=app:CountRandom
     &metric=system.process.events.processed&aggregate=true'
 
 If a metric is a gauge type, the aggregate will return the latest value set for the metric.
 For example, this request will retrieve the completion percentage for the map-stage of the MapReduce
-``PurchaseHistoryWorkflow_PurchaseHistoryBuilder`` (reformatted to fit)::
+``PurchaseHistoryBuilder`` (reformatted to fit)::
 
-  POST '<base-url>/metrics/query?context=namespace.default.app.PurchaseHistory.mapreduce.
-      PurchaseHistoryWorkflow_PurchaseHistoryBuilder&metric=system.process.completion&aggregate=true'
-  
+  POST '<base-url>/metrics/query?tag=namespace:default&tag=app:PurchaseHistory
+    &tag=mapreduce:PurchaseHistoryBuilder&metric=system.process.completion&aggregate=true'
+    
+  {"startTime":0,"endTime":1429497700,"series":[{"metricName":"system.process.completion",
+   "grouping":{},"data":[{"time":0,"value":200}]}]} 
```

Comment:
we are talking about completion percentage and the value is 200 ? that's incorrect.